dprint.json

```diff
@@ -55,4 +56,4 @@
     "https://plugins.dprint.dev/markdown-0.13.3.wasm",
     "https://plugins.dprint.dev/prettier-0.9.0.json@76c20538ea4231b3daa9b62059f90ec144200cab5477b7e95b730cbb3ed48957"
   ]
-}
```

Comment:
@rachfop Ironic that this file had something my editor's autoformatter considered a bad format 😄